### PR TITLE
MOB-372 added global svg icons

### PR DESCRIPTION
**Pull request by lewnelson**
#### What does this PR do? (please provide any background)

This PR takes the mobile icons that were inside holiday extras and moves them over to global where they can then be branded via CSS.

Holiday Extras, LGP, BCP and Airparks brands have all been version bumped to use these new icons. These are the brands which we will be rolling out use of these icons to now and in the near future.

#### How can this be tested?
`cd pattern-library && npm link ../brands`

#### Any tech debt?

N/A

#### What gif best describes how you feel about this work?
Lebowski

- [ ] I have checked our general [contributing document](https://github.com/holidayextras/culture/blob/master/CONTRIBUTING.md) and the project specific [contributing document](../blob/master/CONTRIBUTING.md) (if present) and I'm happy for this to be reviewed.

---

**Reviewer**
- [ ] :+1:
- [ ] UXUI Team :+1:
- [ ] I don't think this PR needs any additional reviewers

By adding a +1 you are confirming you have...
- Witnessed the work behaving as expected (this could be on the author's machine or screencast).
- Ensured the release has the correct version bump.
- Checked all the work locally on pattern library.

### CODE REVIEW

File: brands/globals/src/less/icons.less [hunk 1]
```diff
@@ -39,3 +39,14 @@
   70%  { .scale(.9); }
   100% { .scale(1); }
 });
+
```
Comment (jodiedoubleday) [inline brands/globals/src/less/icons.less]: This is very app and airport specific.
At shortbreaks if we include an SVG with class `.brand-primary` then we want it to be brand primary and not an arbitrary tint of that color.

We need to be mindful that making change in globals could have affect on multiple brands

**Comment by jodiedoubleday** [comment]
While we may need to stop duplication of code/icons in multiple brands i have concerns here that we are adding very airport specific icons and modifying globals less files when 2/3rds of the brands in this repo aren't airport based.
incluing of assets isn't a problem, but modifying global styles is.

CC: @elliottcrush @stuartgiles

**Comment by stuartgiles** [comment]
Just seen @jodiedoubleday comment.
I agree, this smells very specific to TripApp.
Can it be moved? Unless you know of another App that needs them?

@elliottcrush : any thoughts about TripApp specific assets that need to be shared across multiple brands.

File: brands/globals/src/less/icons.less (hunk 1 above)
Comment (lewnelson) [reply brands/globals/src/less/icons.less]: How would you feel about `.main-fill-color` and `.secondary-fill-color` instead?

**Comment by lewnelson** [comment]
@jodiedoubleday @stuartgiles @elliottcrush could the styling from global brands be moved to each individual brand using the icons instead? @stuartgiles we will be using these icons in the same way in render as well. We've got work coming up which is going to be implementing our designs in tripapp into render.

**Pull request merged**